pywbem/cim_obj.py

```diff
@@ -561,7 +565,8 @@ def _makequalifiers(qualifiers, indent):
     """
     if not qualifiers:
         return ''
-    qual_list = [q.tomof(indent + 2) for q in sorted(qualifiers.values())]
+    qual_list = [qualifiers[qn].tomof(indent + 2)
```

Comment:
Not arguing but what is the advantage of the replacement?

Reply:
The sorting was previously done on the dictionary values, i.e. on a list. That uses the ordering comparisons between the list items, i.e. CIMQualifierDeclaration objects. These objects issue a deprecation warning if ordering comparisons are used on them. The change sorts on the dictionary keys and therefore avoids the deprecation warning.